lib/src/protos/op_store.proto

```diff
@@ -16,15 +16,25 @@ syntax = "proto3";
 
 package op_store;
 
+message RefConflictLegacy {
+  repeated bytes removes = 1 [deprecated = true];
+  repeated bytes adds = 2 [deprecated = true];
+}
+
 message RefConflict {
-  repeated bytes removes = 1;
-  repeated bytes adds = 2;
+  message Id {
```

Comment:
nit: It might be better to call this `Term` to match the terminology in e.g. `conflicts.rs`. I think it makes more sense for a "term" to have an optional value than for an "id".

Reply:
Renamed. I also don't like `Id`, but couldn't find a better name.